CRM/Contribute/Form/AbstractEditPayment.php

```diff
@@ -249,6 +249,10 @@ public function __get($name) {
       CRM_Core_Error::deprecatedWarning('attempt to access undefined property _params - use externally supported function getSubmittedValues()');
       return $this->getSubmittedValues();
     }
+    if ($name === '_lineItem') {
+      CRM_Core_Error::deprecatedWarning('attempt to access undefined property _params - use externally supported function getSubmittedValues()');
+      return [$this->getPriceSetID() => $this->getLineItems()];
```

Comment:
... but this shim returns an associative array indexed by price set id. Which one is it?

Reply:
@colemanw hmm - standard behaviour (to the extent it existed) was the price set ID - but you are right - 0 looks more appropriate on this form - I will fix